controllers/complianceeventsapi/server.go

```diff
@@ -92,6 +181,357 @@ func (s *complianceAPIServer) Stop() error {
 	return nil
 }
 
+// splitQueryValue will parse a string and split on unescaped commas. Empty values are discarded.
+func splitQueryValue(value string) []string {
```

Comment:
Note that a normal string split would work just fine in this case because no sort options have a comma in them but I plan to reuse this in my next PR for filter query arguments.